css/style.css

```diff
@@ -0,0 +1,544 @@
+  justify-content: space-around;
+  align-content: center;
+  align-items: start;
+  max-height: 300px;
+}
+
+.level {
+  position: relative;
+  display: block;
+  margin-bottom: 24px;
+  font-size: 48px;
+  line-height: 59px;
+  color: #FFFFFF;
+  margin-left: 46px;
+}
+
+.lvl-active {
+  position: relative;
+}
+
+.lvl-active::after {
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #152fa1;
+  transform: rotate(45deg);
+}
+
+.level::before {
+  display: none;
+  content: '';
+  position: absolute;
+  top: 20px;
+  right: 102%;
+  width: 18px;
+  height: 18px;
+  background: #D62B2E;
+  transform: rotate(45deg);
+}
+
+.level:hover::before {
+  display: block;
+}
+
+.first-card {
+  position: absolute;
+  left: 60%;
+  top: 35%;
+  transform: rotate(11.9deg);
+  z-index: -2;
+}
+
+.second-card {
+  position: absolute;
+  left: 69%;
+  top: 45%;
+  transform: rotate(39.25deg);
+  z-index: -1;
+}
+
+.hide {
+  display: none;
+}
+
+.table {
+  display: none;
+  width: 100%;
+}
+
+.table-0 {
+  width: 720px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.table-1 {
+  display: flex;
+  width: 720px;
+  height: 720px;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.table-2 {
+  display: flex;
+  width: 1200px;
+  height: 720px;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+
+
+
+.card-wrapper {
+  width: 218px;
+  height: 341px;
+  text-align: center;
+  perspective: 1200;
+  -webkit-perspective: 1200;
+  -moz-transform: perspective(1200px);
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d; 
+}
+
+.card-inner {
+  position: relative;
+  width: 100%;
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d;
+  transition: .7s linear;  
+  -webkit-transition: .7s linear;
+  -moz-transition: .7s linear;
+  -o-transition: .7s linear;
+}
+
+.front-side, .back-side {
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 341px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  -webkit-backface-visibility: hidden;
+  -moz-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.back-side {
+  background-image: url(../img/back-side.png);
+  -webkit-transform: rotateY(0deg);
+  -moz-transform: rotateY(0deg);
+  -ms-transform: rotateY(0deg);
+  transform: rotateY(0deg);
+}
+
+.front-side {
+  background-image: url(../img/game-over.png);
+  -webkit-transform: rotateY(-180deg);
+  -moz-transform: rotateY(-180deg);
+  -ms-transform: rotateY(-180deg);
+  transform: rotateY(-180deg);
+}
+
+.front-side--win {
+  background-image: url(../img/winner.png)
+}
+
+.flip {
+  -webkit-transform: rotateY(180deg);
+  -moz-transform: rotateY(180deg);
+  -ms-transform: rotateY(180deg);
+  transform: rotateY(180deg);
+}
+
+@keyframes rotation {
+  from {transform: scale(1);}
```

Comment:
Codacy found an issue: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)